packages/admin/src/middleware.page.ts

```diff
@@ -59,7 +59,7 @@ export async function middleware(req: NextRequest) {
   let url = getLoginUrl();
   console.log('Middleware redirect URL: ' + url);
   if (submissionDownloadPattern.test({ pathname: req.nextUrl.pathname })) {
-    url = url + req.nextUrl.pathname;
+    url = url + `?redirectUrl=${process.env.HOST}` + req.nextUrl.pathname;
```

Comment:
```suggestion
    url = `${url}?redirectUrl=${process.env.HOST}${req.nextUrl.pathname}`;
```

Reply:
realised it and fixed it before seing this 😆